Pod/Classes/PagingMenuOptions.swift

```diff
@@ -28,11 +32,12 @@ public class PagingMenuOptions {
     public var menuItemMode = MenuItemMode.Underline(height: 3, color: UIColor.blueColor(), horizontalPadding: 0, verticalPadding: 0)
     public var lazyLoadingPage: LazyLoadingPage = .Three
     public var menuControllerSet: MenuControllerSet = .Multiple
+    public var menuItemViewContent: MenuItemViewContent = .Text
```

Comment:
Please change to `internal`. 
It's automatically set inside `willSet`, so no need for user to explicitly set it. 
https://github.com/kitasuke/PagingMenuController/pull/207/files#diff-ddd20afa2d86943bffcfe84300a5cfcdR33